src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_PatternSwitchStatement.cs

```diff
@@ -320,7 +320,7 @@ private void AddBindings(ArrayBuilder<BoundStatement> sectionBuilder, ImmutableA
                 {
                     var source = kv.Key;
                     var dest = kv.Value;
```

Comment:
Both are required to be of a form that no further local lowering is possible.

---
In reply to: [80800519](https://github.com/dotnet/roslyn/pull/13999#discussion_r80800519) [](ancestors = 80800519)